grass7/raster/r.width.funct/r.width.funct.py

```diff
@@ -113,6 +110,10 @@ def main():
     print('Done!')
 
 def plotImage(x,y,image,type,xlabel,ylabel,title):
+    import matplotlib #required by windows
+    matplotlib.use('WXAgg') #required by windows
```

Comment:
```suggestion
    matplotlib.use('wxAGG') #required by windows
```

Reply:
Yes, seems to be correct, see https://matplotlib.org/3.2.1/tutorials/introductory/usage.html#the-builtin-backends

Reply:
Anyway, "The names of builtin backends case-insensitive;"...